ui/src/app/shared/models/upload.model.ts

```diff
@@ -12,7 +12,7 @@ export class Upload extends Base implements PageObject {
   public latestVersionId: number;
 
   @serializable(optional())
-  public supportedDeviceType: SupportedDeviceType = SupportedDeviceType.IOS_DEVICE;
+  public supportedDeviceType: SupportedDeviceType = SupportedDeviceType.NULL;
```

Comment:
@nixonakash01 Is there a need for initializing it to SupportedDeviceType.NULL ? What if you initialize it without an explicit NULL value ?

Reply:
It is a enum, so some value is to be selected while initialisation. If we choose ios_device or ios_emulator, the respective radio button is selected. That's why I added a null value initially do that no radio button is selected initially.